File: 2026-01/src/day_08/boj_5596/Main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int sum1 = 0;
    for (int i = 0; i < 4; i++) {
        int x;
        cin >> x;
        sum1 += x;
    }

    int sum2 = 0;
    for (int i = 0; i < 4; i++) {
        int x;
        cin >> x;
        sum2 += x;
    }
    cout << max(sum1, sum2);
}

File: 2026-01/src/day_08/boj_5596/Main.java
package day_08.boj_5596;

import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;

        int sum1 = 0;
        st = new StringTokenizer(br.readLine());
        for (int i = 0; i < 4; i++) {
            sum1 += Integer.parseInt(st.nextToken());
        }

        int sum2 = 0;
        st = new StringTokenizer(br.readLine());
        for (int i = 0; i < 4; i++) {
            sum2 += Integer.parseInt(st.nextToken());
        }

        System.out.println(Math.max(sum1, sum2));
    }
}
